### PR TITLE
fix #77301 desired.de

### LINKED ISSUE #77301: www.desired.de
### Issue URL (Annoyance)
 [https://www.desired.de/lifestyle/lustige-vornamen-die-es-wirklich-gibt/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.desired.de%2Flifestyle%2Flustige-vornamen-die-es-wirklich-gibt%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard for Mac v2.5.3.955 release
Browser: | Safari
Stealth mode options: | Hide your search queries, <br>Send Do-Not-Track header, <br>Remove X-Client-Data header from HTTP requests, <br>Strip URLs from tracking parameters, <br>Self-destructing first-party cookies (4320), <br>Self-destructing third-party cookies (180), <br>Block Push API, <br>Block Location API, <br>Block Java
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Other:</b><br/> AdGuard DNS<br/><br/><b>Language-specific:</b><br/> AdGuard Russian, <br/>AdGuard German, <br/>AdGuard Japanese, <br/>AdGuard Dutch, <br/>AdGuard Spanish/Portuguese, <br/>AdGuard French, <br/>Bulgarian list, <br/>EasyList Czech and Slovak, <br/>EasyList Hebrew, <br/>EasyList Italy, <br/>EasyList Lithuania, <br/>Icelandic ABP List, <br/>Hungarian

### PR DESCRIPTION
**Pull request by wout-codes**
#77301

### CODE REVIEW

File: AnnoyancesFilter/sections/cookies_specific.txt [hunk 1]
```diff
@@ -86,7 +86,7 @@ sportbuzzer.de,fitbook.de,t-online.de,wunderweib.de,hitc.com,haz.de,rnd.de,faz.n
...
+/wrapperMessagingWithoutDetection.js$script,domain=sportbible.com|vox.de|essen-und-trinken.de|jameda.de|gutefrage.net|planetiphone.de|computerfrage.net|helpster.de|heise.de|dpd.com|dpd.de|duden.de|sky.de|sky.com|tvmovie.de|cardscout.de|meineorte.com|tecchannel.de|itpro.co.uk|tvnow.at|tvnow.de|netdoktor.de|computerwoche.de|the-sun.com|thescottishsun.co.uk|thesun.co.uk|thesun.ie|handelsblatt.com|macwelt.de|lancashiretelegraph.co.uk|idealo.de|lvz.de|myhomebook.de|travelbook.de|4players.de|n-page.de|planet3ds.de|planetds.de|planetgameboy.de|planetswitch.de|planetvita.de|playstationportable.de|portablegaming.de|gala.de|denofgeek.com|classicfm.com|globalplayer.com|radio.pl|politico.eu|expertreviews.co.uk|radio.net|rtlplus.de|rtl-passion.de|geo-television.de|desired.de
```
Comment (Alex-302) [inline AnnoyancesFilter/sections/cookies_specific.txt:89]: it breaks player at the bottom. 
Also misset some stuff.

Comment (wout-codes) [reply AnnoyancesFilter/sections/cookies_specific.txt:89]: Yeah, didn't notice. Let me check.

Comment (wout-codes) [reply AnnoyancesFilter/sections/cookies_specific.txt:89]: @Alex-302 `desired.de#%#//scriptlet("set-constant", "consentIsActive", "false")`

Comment (wout-codes) [reply AnnoyancesFilter/sections/cookies_specific.txt:89]: `desired.de#%#//scriptlet("set-constant", "consentIsActive", "false")` fixes the player if added with `/wrapperMessagingWithoutDetection.js$script,domain=sportbible.com|vox.de|essen-und-trinken.de|jameda.de|gutefrage.net|planetiphone.de|computerfrage.net|helpster.de|heise.de|dpd.com|dpd.de|duden.de|sky.de|sky.com|tvmovie.de|cardscout.de|meineorte.com|tecchannel.de|itpro.co.uk|tvnow.at|tvnow.de|netdoktor.de|computerwoche.de|the-sun.com|thescottishsun.co.uk|thesun.co.uk|thesun.ie|handelsblatt.com|macwelt.de|lancashiretelegraph.co.uk|idealo.de|lvz.de|myhomebook.de|travelbook.de|4players.de|n-page.de|planet3ds.de|planetds.de|planetgameboy.de|planetswitch.de|planetvita.de|playstationportable.de|portablegaming.de|gala.de|denofgeek.com|classicfm.com|globalplayer.com|radio.pl|politico.eu|expertreviews.co.uk|radio.net|rtlplus.de|rtl-passion.de|geo-television.de|desired.de`

Comment (Alex-302) [reply AnnoyancesFilter/sections/cookies_specific.txt:89]: It is not required now, but it looks useful.

Comment (wout-codes) [reply AnnoyancesFilter/sections/cookies_specific.txt:89]: I will add it yeah

Comment (wout-codes) [reply AnnoyancesFilter/sections/cookies_specific.txt:89]: I will remove it from wrapper message, it will break the player for iOS users and CB.

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**